src/screens/Terms/Terms.tsx

```diff
@@ -55,7 +55,7 @@ const Terms = ({ children }: { children: React.ReactNode }) => {
           Website Terms of Use
         </Typography>
         <Typography variant="h6" component="h1" align="center">
-          Last updated: October 9, 2020
+          Last updated: October 26, 2022
```

Comment:
TBH I wouldn't bump the last updated date for this.  It seems like a completely immaterial change.  I don't feel strongly though.